src/index.scss

```diff
@@ -0,0 +1,231 @@
+@import './variables';
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: $font-family-roboto;
+}
+
+header {
+  box-shadow: 0 2px 4px 0 #0000000d;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $white;
+  padding: 0 50px;
+  height: 60px;
+
+  .header__logo {
+    img {
+      height: 40px;
+    }
+  }
```

Comment:
The class `.header__logo img` should be reviewed for BEM consistency. Consider using a more descriptive name like `.header__logo-image` to align with BEM principles.